README.md

```diff
@@ -87,6 +90,7 @@ local function _burn(from, amount)
 -- @param   to      (address) recipient's address
 -- @param   amount  (ubig) amount of tokens to send
 -- @param   ...     addtional data, MUST be sent unaltered in call to 'tokensReceived' on 'to'
```

Comment:
Instead of "MUST be sent ..." it could be `additional data, is sent unaltered in call to 'tokensReceived' on 'to'`